dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -340,16 +340,17 @@ public static JsonNode dockstoreEntryToElasticSearchObject(final Entry entry) th
         objectNode.put("stars_count", (long) entry.getStarredUsers().size());
         objectNode.put("verified", verified);
         objectNode.put("openData", openData);
-        objectNode.put("verified_platforms", MAPPER.valueToTree(verifiedPlatforms));
-        objectNode.put("descriptor_type_versions", MAPPER.valueToTree(descriptorTypeVersions));
-        objectNode.put("engine_versions", MAPPER.valueToTree(engineVersions));
-        objectNode.put("all_authors", MAPPER.valueToTree(allAuthors));
-        objectNode.put("categories", MAPPER.valueToTree(convertCategories(entry.getCategories())));
+        objectNode.set(EXECUTION_PARTNERS, MAPPER.valueToTree(entry.getExecutionPartners()));
```

Comment:
Used `set` and changed existing `put`s to `set` to avoid SonarCloud flagged deprecation warnings. You still use `put` if the second argument is not a `JsonNode`, which is why there are still `put`s in this code block.